config.c

```diff
@@ -508,6 +508,13 @@ static bool apply_style_option(struct mako_style *style, const char *name,
 	return false;
 }
 
+bool apply_global_option(struct mako_config *config, const char *name,
+		const char *value) {
+	struct mako_criteria *global = global_criteria(config);
```

Comment:
Applying these to the global (first) criteria means you'll only be able to override style options that aren't set by anything in the config file. I'm not sure this is what most users would expect. For example, the do not disturb use case won't work with my config because I make "clever" use of invisible and group-index.

On the other hand, if we inserted a criteria last in the order for IPC to update, `invisible=false` over IPC would break hiding grouped notifications for the default config.

All that is to say, I think updating the global criteria is the right choice for now. Just wanted to get this out there to think about. Not sure what the right solution is. (For do not disturb specifically, we may just want a new config option.)